lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/form/Auto.js

```diff
@@ -127,7 +127,7 @@ qx.Class.define("callbackery.ui.form.Auto", {
             if (s.set) {
                 if (s.widget == 'date') {
                     if (s.set.value) {
-                        var dateValue = s.set.value;
+                        var dateValue = s.set.value * 1000; // incoming epoch seconds
                         delete s.set.value;
```

Comment:
ok ... before it was possible to pass a string which would have initialized the date object too. now input must be an integer ... fine with me